CodeEditModules/Modules/AppPreferences/src/SoftwareUpdater.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  SoftwareUpdater.swift
+//  CodeEdit
+//
+//  Created by Austin Condiff on 9/19/22.
+//
+
+import Foundation
+import Sparkle
+
+public class ObservableUpdater: ObservableObject {
```

Comment:
Should this be named `SoftwareUpdater` like your file name? #picky

Reply:
I thought of that, I was torn because it is an ObservableObject but I wanted to be clear as to what it was if I was putting it in it's own file. I think you are right though. I will change it

Reply:
Done.